src/guide_curves/CTiglCurveConnector.cpp

```diff
@@ -323,7 +323,12 @@ void CTiglCurveConnector::InterpolateGuideCurvePart(guideCurveConnected& connect
 
     GeomAPI_Interpolate interpol(hpoints, hparams, Standard_False, Precision::Confusion());
     interpol.Load(htangents, htangentFlags, false);
-    interpol.Perform();
+    try {
+        interpol.Perform();
+    } catch(...)
```

Comment:
What exception is thrown here? Shouldn't we try to catch a specific exception?

Reply:
I figured it makes sense to convert any thrown exception to a `CTiglException`. The problem was that we get a `Standard_ConstructionError` from OpenCascade which never got caught. CTiglExceptions do get caught.

Reply:
Lets use Standard_Failure to be on the safe side. I can commit this